main.py

```diff
@@ -12,12 +12,13 @@
 import boto3
 from enums.cameraEnums import CameraMode, CameraStatus
 from enums.eventEnums import EventType
+from os import environ
 
 app = Flask(__name__)
 app.config['SECRET_KEY'] = 'imageanalysissystem'
-app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///database/database.db'
-#CONFIG AWS KEYS/BUCKET DETAILS
-#app.config["S3_LOCATION"] = "us-west-2"
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://postgres:' + environ["DB_PASSWORD"] + '@lfiasdb.cwtorsyu3gx6.us-west-2.rds.amazonaws.com/postgres'
```

Comment:
So, is this a **postgres** db now instead of an **SQLite**?

Reply:
So my understanding of this... The way we talk to the DB is going to be the exact same, just the location of our database has now changed to live in AWS instead of locally. Is that correct?

Reply:
Yeah, unfortunately AWS doesn't easily support SQLite

Reply:
Yup exactly, SQLalchemy made it pretty easy, just had to configure the DB properly